packages/codemods/src/transforms/modal-card.ts

```diff
@@ -12,33 +14,43 @@ export default function transformer(file: FileInfo, api: API, options: JSCodeShi
   const { localName: localNameModalCard } = getImportInfo(j, file, 'ModalCard', alias);
   const { localName: localNameModalCardBase } = getImportInfo(j, file, 'ModalCardBase', alias);
 
-  source
+  const modalCards = source
     .find(j.JSXOpeningElement)
     .filter(
       (path) =>
         path.value.name.type === 'JSXIdentifier' &&
         [localNameModalCard, localNameModalCardBase].includes(path.value.name.name),
-    )
-    .forEach((path) => {
-      const headerAttribute = j(path).find(j.JSXAttribute, { name: { name: 'header' } });
-      const subheaderAttribute = j(path).find(j.JSXAttribute, { name: { name: 'subheader' } });
-
-      if (subheaderAttribute.length > 0) {
-        if (path.node.attributes) {
-          path.node.attributes.push(
-            j.jsxAttribute(j.jsxIdentifier('subheaderComponent'), j.stringLiteral('h5')),
-          );
-        }
+    );
+
+  modalCards.forEach((path) => {
+    const headerAttribute = j(path).find(j.JSXAttribute, { name: { name: 'header' } });
+    const subheaderAttribute = j(path).find(j.JSXAttribute, { name: { name: 'subheader' } });
+
+    if (subheaderAttribute.length > 0) {
+      if (path.node.attributes) {
+        path.node.attributes.push(
+          j.jsxAttribute(j.jsxIdentifier('subheaderComponent'), j.stringLiteral('h5')),
+        );
       }
+    }
 
-      if (headerAttribute.length > 0) {
-        if (path.node.attributes) {
-          path.node.attributes.push(
-            j.jsxAttribute(j.jsxIdentifier('headerComponent'), j.stringLiteral('h2')),
-          );
-        }
+    if (headerAttribute.length > 0) {
+      if (path.node.attributes) {
+        path.node.attributes.push(
+          j.jsxAttribute(j.jsxIdentifier('headerComponent'), j.stringLiteral('h2')),
+        );
       }
-    });
+    }
+  });
+
+  if (modalCards.size() > 0) {
+    report(
+      api,
+      `: ${chalk.white.bgBlue('ModalCard')} and ${chalk.white.bgBlue(
+        'ModalCardBase',
+      )} might have 'Spacing' now. Manual changes required.`,
```

Comment:
Не настаиваю, но может быть стоит сказать поконкретнее.
```suggestion
      )} might need 'Spacing' now. See migration guide for more details. Manual changes required.`,
```

Reply:
`have` на `need` поменяла, а про гайд (с ссылкой на него), мы по дефолту к каждому сообщения лепим х)

Reply:
Ага, спасибо!

>  а про гайд (с ссылкой на него), мы по дефолту к каждому сообщения лепим х)

Я подозревал 🙂 👍